### ISSUE TITLE
Pin message

### ISSUE THREAD

**Issue by Slavenin** [zhyu/nadia#94]
How i can pin message with bot help? 
https://core.telegram.org/bots/api#pinchatmessage

**Comment by m1ome** [comment]
@Slavenin right now you can use (before it would be merged)
```elixir
Nadia.API.request("pinMessage", [chat_id: 1, message_id: 1])
```

**Comment by zhyu** [comment]
I just merged #96 , you can use it if you install `nadia` from github.
I'll release a new version later.
